src/styles/components/calendar.scss

```diff
@@ -0,0 +1,55 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: $maxWidth;
+  gap: #{$gap};
+
+  &--month-length--29 {
+    margin-top: 50px;
+  }
```

Comment:
I got your idea but let me speak straight - this solution is even worse because it 1) does not work for calendars of 28, 30, and 31 days and 2) it still has the same problem of the margin applied to the wrong place